hotness/consumers.py

```diff
@@ -297,8 +351,8 @@ def handle_buildsys_scratch(self, msg):
         text2 = "%s %s" % (subtitle, url)
 
         # Followup on bugs we filed
-        if task_id in self.triggered_task_ids:
-            bugs.append((self.triggered_task_ids.pop(task_id), text1))
+        #if task_id in self.triggered_task_ids:
+        #    bugs.append((self.triggered_task_ids.pop(task_id), text1))
```

Comment:
Yeah, right here is where hotness would pick back up on a task that it kicked off previously.